task_02/index.html

```diff
@@ -1,11 +1,194 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
 </head>
+
 <body>
-    Task 2
+  <div class="wrapper">
+    <!-- Desktop navigation -->
+    <nav role="navigation" class="desktop-navigation">
+      <div class="nav-wrapper">
+        <div class="list-item"><a href="" class="links">Home</a></div>
+        <div class="list-item"><a href="" class="links">Approach</a></div>
+        <div class="list-item"><a href="" class="links">Work</a></div>
+        <div class="list-item"><a href="" class="links">People</a></div>
+        <div class="list-item">
+          <a href="" class="links" id="disabled">Jobs</a>
+        </div>
+        <div class="list-item">
+          <a href="" class="links" id="disabled">Blog</a>
+        </div>
+        <div class="list-item">
+          <a href="" class="links" id="disabled">Contact</a>
+        </div>
+      </div>
+    </nav>
+
+    <!-- Mobile nav -->
+    <nav role="navigation" class="mobile-navigation">
+      <div id="menuToggle">
+        <input type="checkbox" id="toggleCheckbox" onclick="moveText()">
+
+        <span></span>
+        <span></span>
+        <span></span>
+
+        <ul id="menu">
+          <a href="#">
+            <li>Home</li>
+          </a>
+          <a href="#">
+            <li>Approach</li>
+          </a>
+          <a href="#">
+            <li>Work</li>
+          </a>
+          <a href="#">
+            <li>People</li>
+          </a>
+          <a href="#" class="disabled">
+            <li>Jobs</li>
+          </a>
+          <a href="#" class="disabled">
+            <li>Blog</li>
+          </a>
+          <a href="#" class="disabled">
+            <li>Contact</li>
+          </a>
+        </ul>
+      </div>
+    </nav>
+
+    <!-- Other content -->
+    <section id="section">
+      <div class="item" id="item1">
+        <div class="section-wrapper">
+          <div class="text-wrapper">
```

Comment:
Do we need to divs `section-wrapper` and `text-wrapper`? As I can see it's just div inside div without no reason? Am I wrong?

Reply:
We need,please let me explain this. Since we have section-wrapper div with : 
`display:flex` along with `align-items: center` and `justify-content: center` it refers also on nested div which is in our case text-wrapper so all content inside of it will be centered. If there is no text-wrapper around these two text divs inside of it this wouldn't look like this. It will look like first text will be centered but at left part of div and second will be on right part and that is something that we don't want to happen so this wrapper is one of solutions to prevent this to happen.